ariadne_codegen/client_generators/result_types.py

```diff
@@ -163,6 +174,26 @@ def generate(self) -> ast.Module:
             )
         return module
 
+    def include_model_rebuild(self, class_def: ast.ClassDef) -> bool:
+        class NameVisitor(ast.NodeVisitor):
```

Comment:
Why declare this class within a method?

Reply:
Initially I wanted to add at the top of the file, but I suggested https://github.com/mirumee/ariadne-codegen/blob/ac92bd1f7d818d5cde79e389318d1761787e29f4/ariadne_codegen/client_generators/result_types.py#L582-L589

Reply:
But this declaration is a closure for `node` and yours isn't.

Reply:
ok, what solution do you suggest ?

Reply:
Move it under generator class.